tokio-macros/src/lib.rs

```diff
@@ -423,6 +463,47 @@ pub fn main_rt(args: TokenStream, item: TokenStream) -> TokenStream {
 ///     println!("Hello world");
 /// }
 /// ```
+///
+/// ### Configure unhandled panic behavior
+///
+/// Available options are `shutdown_runtime` and `ignore`. For more details, see
+/// [`Builder::unhandled_panic`].
+///
+/// This option is only compatible with the `current_thread` runtime.
+///
+/// ```ignore
```

Comment:
For consistency with the other examples on this page, can you use `no_run` instead of `ignore`?
(The `ignore` annotation makes the doc a bit scary.)
<img width="262" alt="Screen Shot 2024-06-03 at 19 57 29" src="https://github.com/tokio-rs/tokio/assets/55653825/ebb130a5-7d7f-476a-9c69-4d29b93d137f">

Reply:
> For consistency with the other examples on this page, can you use `no_run` instead of `ignore`? (The `ignore` annotation makes the doc a bit scary.) <img alt="Screen Shot 2024-06-03 at 19 57 29" width="262" src="https://private-user-images.githubusercontent.com/55653825/336044251-ebb130a5-7d7f-476a-9c69-4d29b93d137f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1OTg3MzAsIm5iZiI6MTcxNzU5ODQzMCwicGF0aCI6Ii81NTY1MzgyNS8zMzYwNDQyNTEtZWJiMTMwYTUtN2Q3Zi00NzZhLTljNjktNGQyOWI5M2QxMzdmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjA1VDE0NDAzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFlZWI1ZTZmMTlhNTE3ZDI0MTU3NTUzZmFjZjhkMzMyODEyMjViYTVlOTdkNTFhZTQxNWQ0OGM3M2RhYTk5MjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.sKR3OQmZI3I3ziUfwhxB5s5gRM4MeE8YcdvZ1tzJASI">

I tried and found it break those CI jobs running `cargo test` without `RUSTFLAGS="--cfg tokio_unstable"`, such as [this link](https://github.com/tokio-rs/tokio/actions/runs/9385991492/job/25845581971?pr=6593). Since we can't use conditional compilation tricks on examples, I think we can only use `ignore` here to make CI happy.

Reply:
I'm pretty sure that `#[cfg(tokio_unstable)]` works perfectly fine inside an example.

Reply:
> I'm pretty sure that `#[cfg(tokio_unstable)]` works perfectly fine inside an example.

Yes, it works fine. I mean we can not use words like
```
```#[cfg(tokio_unstable)]
```
to disable an example entirely. But your comment reminds me that we can use `#` lines to make the example works under `cargo test` without `tokio_unstable `, I will try it.

Reply:
> > I'm pretty sure that `#[cfg(tokio_unstable)]` works perfectly fine inside an example.
> 
> Yes, it works fine. I mean we can not use words like
> 
> ```
> ```#[cfg(tokio_unstable)]
> ```
> 
> to disable an example entirely. But your comment reminds me that we can use `#` lines to make the example works under `cargo test` without `tokio_unstable `, I will try it.

All tests looks fine now.